examples/manifests/zarf.yaml

```diff
@@ -3,6 +3,10 @@ metadata:
   name: manifests
   version: 0.0.1
 
+variables:
+  - name: HTTPD_VERSION
+    default: "3.18"
```

Comment:
This isn't a great example since this version might not be in the package - it would be better to have a variable for a different value (potentially something that enables/disables a dependency in a chart)